src/Tablero.cpp

```diff
@@ -7,21 +7,35 @@
 
 #include "Tablero.h"
 
-Tablero::Tablero(string nombre, int filas, int columnas) {
+void Tablero::inicializar(std::string nombre, int filas, int columnas) {
 	this->nombre = nombre;
 	this->filas = filas;
 	this->columnas = columnas;
-	this->tablero = new Parcela*[filas];
-	for(int i=0; i<filas; i++){
-		this->tablero[i] = new Parcela[columnas];
+	this->parcelas = new Parcela*[this->filas];
+	for(int i = 0; i < this->filas; i++){
+		this->parcelas[i] = new Parcela[columnas];
```

Comment:
te parece hacer Parcela[this->columnas] ya que usamos this->filas? por consistencia